### PR TITLE
add request comics by series.

### PR DESCRIPTION
**Pull request by flipper83**
add request comics by series.

### CODE REVIEW

File: MarvelApiClient/src/main/java/com/karumi/marvelapiclient/SeriesApiClient.java [hunk 1]
```diff
@@ -65,4 +67,23 @@ public SeriesApiClient(MarvelApiConfig marvelApiConfig) {
...
     }
   }
+
+  public MarvelResponse<ComicsDto> getComicsBySerie(String seriesId, int offset, int limit)
+      throws MarvelApiException {
+    ComicsQuery query = ComicsQuery.Builder.create().withOffset(offset).withLimit(limit).build();
+    return getComicsBySerie(seriesId, query);
+  }
+
+  public MarvelResponse<ComicsDto> getComicsBySerie(String seriesId, ComicsQuery comicsQuery)
```
Comment (pedrovgs) [inline MarvelApiClient/src/main/java/com/karumi/marvelapiclient/SeriesApiClient.java]: By serieSSSSS

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java [hunk 2]
```diff
@@ -66,6 +67,18 @@
   private static final String ANY_NOT_FOUND_ID = "1234";
   private static final String ANY_SERIE_ID = "123456";
   public static final String EVENTS_REQUEST = "1,2";
+  private static final int ANY_SERIE = 1;
+  private static final int ANY_SHARED_APPEARANCES = 4;
+  public static final String COLLABORATORS_REQUEST = "1,2";
```
Comment (pedrovgs) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java]: private

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java [hunk 3]
```diff
@@ -184,6 +197,77 @@ public void shouldReturnMarvelExceptionWhenTheIdDoesNotExist() throws Exception
     assertIs5Ronnin(response);
   }
 
+  @Test public void shouldSendGetComicBySeriesRequestToTheCorrectEndpoint() throws Exception {
+    SeriesApiClient seriesApiClient = givenSeriesApiClient();
+    enqueueMockResponse("getComics.json");
```
Comment (pedrovgs) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java:202]: Shouldn't this one be getComicsBySeriesId.json?

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java [hunk 4]
```diff
@@ -184,6 +197,77 @@ public void shouldReturnMarvelExceptionWhenTheIdDoesNotExist() throws Exception
...
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void shouldThrowExceptionWhenCallGetComicBySerieAndSerieIdIsEmpty() throws Exception {
+    SeriesApiClient seriesApiClient = givenSeriesApiClient();
+    enqueueMockResponse();
+
+    seriesApiClient.getComicsBySerie("", ANY_OFFSET, INVALID_LIMIT);
+  }
+
```
Comment (pedrovgs) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java:263]: Don't you have a test to check the parsing for comics by series is working?

File: MarvelApiClient/src/main/java/com/karumi/marvelapiclient/SeriesApiClient.java (hunk 1 above)
Comment (flipper83) [inline MarvelApiClient/src/main/java/com/karumi/marvelapiclient/SeriesApiClient.java]: resucks

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java (hunk 2 above)
Comment (flipper83) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java]: yep

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java (hunk 3 above)
Comment (flipper83) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java:202]: the response it's the same that getComics, for not repeat the test.

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java (hunk 4 above)
Comment (flipper83) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java:263]: the response it's the same that getComics, for not repeat the test.

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java (hunk 3 above)
Comment (pedrovgs) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java:202]: ok

File: MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java (hunk 4 above)
Comment (pedrovgs) [inline MarvelApiClient/src/test/java/com/karumi/marvelapiclient/SeriesApiClientTest.java:263]: ok

**Comment by pedrovgs** [comment]
:+1:

**Pull request merged**